{"name": "Не успешая авторизация", "status": "passed", "steps": [{"name": "Нажать Аккаунт", "status": "passed", "start": 1684322273836, "stop": 1684322273879}, {"name": "Нажать Войти", "status": "passed", "start": 1684322273879, "stop": 1684322274027}, {"name": "Войти как 'qwerty@qwerty.qwerty' 'VTer38CX'", "status": "passed", "start": 1684322277036, "stop": 1684322277253}, {"name": "Проверить что ошибка логина отображается", "status": "passed", "start": 1684322277253, "stop": 1684322277843}], "attachments": [{"name": "log", "source": "70cc6cce-ba70-4a9b-a15f-f1865f781cf5-attachment.txt", "type": "text/plain"}], "start": 1684322273836, "stop": 1684322277844, "uuid": "5965fc59-5220-408c-b24b-a4e3ec9404ac", "historyId": "8eaf6323d358d700d14848100a31cbc2", "testCaseId": "7b6113008a0e4d8a5bad2235a270375d", "fullName": "tests.test_login.TestLogin#test_login_fail", "labels": [{"name": "feature", "value": "Тесты логина"}, {"name": "tag", "value": "login"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-VLT8PU4"}, {"name": "thread", "value": "836-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}